toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -911,7 +905,8 @@ func (b *LiveOSIsoBuilder) generateInitrdImage(rootfsSourceDir, artifactsSourceD
 			initrdPathInChroot,
 			"--kver", b.artifacts.kernelVersion,
 			"--filesystems", "squashfs",
-			"--include", artifactsSourceDir, artifactsTargetDir}
+			"--include", artifactsSourceDir, artifactsTargetDir,
+			"--include", "/usr/lib/locale", "/usr/lib/locale"}
```

Comment:
We should really just solve this in the core initramfs configs if possible... (you don't need to fix here though)

Reply:
mm... we call dracut directly here, Daniel though, a fix in the initramfs package will not carry over, correct?

Reply:
Removed this change from the PR.